app/models/setting.rb

```diff
@@ -5,6 +5,9 @@ class Setting < ApplicationRecord
   validates :value, presence: true
   validates :code, uniqueness: true
 
+  scope :voog_site_url, -> { find_by(code: :voog_site_url)&.value }
```

Comment:
I suppose better use class methods here.
You see, main scopes usage is to return `ActiveRecord::Relation`, not an instance of class, and scopes are intended to be linked (chained) to other relations. And here you return not only instance, but an attribute of this instance, it's not so good behaviour preventing us to chain this scope like `Auction.active.last` and so.

Reply:
Makes sense, yup. Making changes.